content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -30,15 +30,23 @@ Use cases for raw SQL include:
 `$queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
 
 ```ts no-lines
-const result = await prisma.$queryRaw('SELECT * FROM User;')
+const result = await prisma.$queryRaw`SELECT * FROM User`
 ```
 
-The same method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which makes it easier to use [variables](#using-variables) by providing **placeholders**. The following example includes an `${email}` placeholder:
+The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which makes it easier to use [variables](#using-variables) by providing **placeholders**. The following example includes an `${email}` placeholder:
 
 ```ts no-lines
-const email = 'edna@prisma.io'
+const email = 'bruce@avengers.io'
```

Comment:
Why switching to Avengers? I'd stick to something as neutral (and diverse) as possible ideally.

Reply:
Yeah, you are 100% correct. I had changed one and thought about changing all the others to be consistent. But its not really the place for it to be fair 😅